py3status/modules/gpmdp.py

```diff
@@ -4,55 +4,125 @@
 
 Configuration parameters:
     cache_timeout:  refresh interval for this module (default 5)
-    format:         specify the items and ordering of the data in the status bar.
-                    These area 1:1 match to gpmdp-remote's options
-                    (default '♫ {info}')
+    format:         display format for this module (default '♫[ {status}]')
+    state_paused:   show this when gpmdp paused playback
+        (default '[(Paused) {song_title} - {song_artist}]')
+    state_playing:  show this when gpmdp started playback
+        (default '{song_title} - {song_artist}')
```

Comment:
I'm kinda way ahead when playing with this stuffs for `sf-mpd_status` and other `sf-*` branch. I feel that using `state_` is a nice approach (for everything, not just music). We could use `State placeholders` along with `Format placeholders`.

Reason? I even safeformatted the separators and every little thing I could find... so we basically have `format_separator` and other similar names... Always starting off with `format_`. Feels old really fast. I think we should just safeformat everything and not worrying about `format_` prefix. We leave `format` alone. Idk. Something to consider.